test/static/async-e2e-page.html

```diff
@@ -12,7 +12,7 @@
     <script type="text/javascript">
       window.addEventListener('load', () => {
         const intakeOrigin = `http://${window.location.hostname}:4000`
-        const specIdParam = /spec-id=\d+/.exec(window.location.search)[0]
+        const specIdParam = window.location.search ? /spec-id=\d+/.exec(window.location.search)[0] : ''
```

Comment:
```suggestion
        const specIdParam = /spec-id=\d+/.exec(window.location.search) && /spec-id=\d+/.exec(window.location.search)[0]
```
it would avoid unexpected behavior if at some point there is something in search that is not spec-id